web/storybook.tiltfile

```diff
@@ -1,6 +1,10 @@
 # Enforce versioning so that labels are supported
 version_settings(constraint='>=0.22.1')
 
+# Uncomment to try the cancel button extension
+#v1alpha1.extension_repo(name='default', url='file:///home/nick/src/tilt-extensions')
```

Comment:
Harmless but just commenting in case you meant to remove

Reply:
nope! just showing what this looks like. the v1alpha1 in the module name is a bit weird IMHO, but it's a k8s convention that i guess people are familiar with? and will put pressure on us to make it v1